### PR TITLE
Update introduction page

### PR DESCRIPTION
**Pull request by havan**
This PR:

- Updates intro page with revisions from @vovanovaque.
- Uses name "Camino Network" more consistently in the related sections.
- Fixes old Twitter & foundation URLs.

### CODE REVIEW

File: docs/developer/apis/camino-node-apis/index-api.md [hunk 1]
```diff
@@ -394,7 +396,7 @@ To get an X-Chain transaction by its index (the order it was accepted in), use I
 For example, to get the _second_ transaction (note that `"index":1`) accepted on the X-Chain, do:
 
 ```sh
-curl --location --request POST 'https://indexer-test.camino.foundation/ext/index/X/tx' \
+curl --location --request POST 'https://indexer-test.camino.network/ext/index/X/tx' \
```
Comment (mo-c4t) [inline docs/developer/apis/camino-node-apis/index-api.md:399]: there is no such subdomain . maybe it should be https://api.network/ext/index/X/tx  ?

File: docs/developer/apis/camino-node-apis/keystore.md [hunk 2]
```diff
@@ -8,7 +8,7 @@ Every node has a built-in keystore. Clients create users on the keystore, which
 
 _**You should only create a keystore user on a node that you operate, as the node operator has access to your plaintext password.**_
 
-For validation and delegation on main net, you should issue transactions through [the wallet](https://wallet.camino.foundation). That way control keys for your funds won't be stored on the node, which significantly lowers the risk should a computer running a node be compromised.
+For validation and delegation on main net, you should issue transactions through [the wallet](https://wallet.camino.network). That way control keys for your funds won't be stored on the node, which significantly lowers the risk should a computer running a node be compromised.
```
Comment (mo-c4t) [inline docs/developer/apis/camino-node-apis/keystore.md]: we don't use wallet.camino.network. it should be https://suite.camino.network/wallet

File: docs/developer/references/cryptographic-primitives.md [hunk 3]
```diff
@@ -120,7 +120,7 @@ As an example we will sign the message "Through consensus to the stars"
 54 68 72 6f 75 67 68 20 63 6f 6e 73 65 6e 73 75 73 20 74 6f 20 74 68 65 20 73 74 61 72 73
 ```
 
-After hashing with `sha256` and signing the pre-image we return the value [cb58](#cb58) encoded: `4Eb2zAHF4JjZFJmp4usSokTGqq9mEGwVMY2WZzzCmu657SNFZhndsiS8TvL32n3bexd8emUwiXs8XqKjhqzvoRFvghnvSN`. Here's an example using the [Camino Web Wallet](https://wallet.camino.foundation).
+After hashing with `sha256` and signing the pre-image we return the value [cb58](#cb58) encoded: `4Eb2zAHF4JjZFJmp4usSokTGqq9mEGwVMY2WZzzCmu657SNFZhndsiS8TvL32n3bexd8emUwiXs8XqKjhqzvoRFvghnvSN`. Here's an example using the [Camino Web Wallet](https://wallet.camino.network).
```
Comment (mo-c4t) [inline docs/developer/references/cryptographic-primitives.md]: should be suite.camino.network

**Review by mo-c4t — CHANGES_REQUESTED** [review]

File: docs/developer/apis/camino-node-apis/index-api.md (hunk 1 above)
Comment (havan) [reply docs/developer/apis/camino-node-apis/index-api.md:399]: I'll take a look and fix this in another PR

**Review by mo-c4t — APPROVED** [review]

**Pull request merged**